.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -112,15 +123,25 @@ body:
     id: reproduction-steps
     attributes:
       label: Steps to Reproduce
-      description: Providing clear, step-by-step instructions helps us reproduce and fix the issue faster. If we can't reproduce it, we can't fix it.
+      description: |
+        Please provide a **very detailed, step-by-step guide** to reproduce the issue. Your instructions should be so clear and precise that anyone can follow them without guesswork. Include every relevant detail—settings, configuration options, exact commands used, values entered, and any prerequisites or environment variables.  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)